CWE/events/Italy ahistorical.txt

```diff
@@ -0,0 +1,4166 @@
+			relation = { who = THIS value = 50 }
+        }
+		
+		ai_chance = { factor = 0.30 }
+	}
+	
+	option = {
+		name = "Keep the parties as they are"
+		badboy = 3
+		
+		prestige = -25
+		
+		any_pop = { 
+		    ideology = { factor = 0.01 value = nationalist } 
+			ideology = { factor = 0.03 value = conservative } 
+			scaled_consciousness = { ideology = socialist factor = 4 }
+			scaled_militancy = { ideology = socialist factor = 4 }
+			scaled_consciousness = { ideology = communist factor = 4 }
+			scaled_militancy = { ideology = communist factor = 4 }
+			scaled_consciousness = { ideology = communist_social factor = 4 }
+			scaled_militancy = { ideology = communist_social factor = 4 }
+			consciousness = 2
+			militancy = 2
+		}
+		random_country = {
+			limit = { tag = USA OR = { government = democracy government = democracy1 government = presidential_dictatorship } is_greater_power = yes }
+			relation = { who = THIS value = 75 }
+        }
+		
+		clr_country_flag = ITA_Democratic_Opposition
+		clr_country_flag = ITA_Democracy_Elections2
+		set_country_flag = ITA_Regime_coninuity
+		set_country_flag = ITA_Democracy_Elections
+		ai_chance = { factor = 0.05 }
+	}
+	
+}
+
+#Elections 4
+country_event = {
+ 	id = 8020417
+  	title = "$YEAR$ Italian General Election"
+ 	desc = EVT_8020417_DESC
+  	picture = "ITA_1953_elections"
+
+	trigger = {
+		tag = ITA
+		NOT = { has_global_flag = 1992_start_date_flag }
+		
+		government = democracy
+		is_vassal = no
+		war = no
+		
+		OR = { ruling_party_ideology = liberal ruling_party_ideology = socialist ruling_party_ideology = conservative }
+		
+		has_country_flag = ITA_Democracy_Restored
+		has_country_flag = ITA_Democracy_Elections2
+		has_country_flag = ITA_Democratic_Opposition
+		has_global_flag = ITA_Operation_Tora_Tora
+	}
+	
+	mean_time_to_happen = { months = 60 }
+	
+	immediate = { clr_country_flag = ITA_Democracy_Elections2 set_country_flag = ITA_Democracy_Elections }
+	
+	option = {
+		name = "New Centre (PRI-PSDI)"
+		ruling_party_ideology = liberal
+		
+		any_pop = {
+			ideology = { factor = 0.07 value = big_tent }
+			ideology = { factor = 0.15 value = liberal }
+		}
+		
+		ai_chance = { factor = 0.6 }
+	}
+	
+	option = {
+		name = "Left Cartel (PSI-PR)"
+		ruling_party_ideology = socialist
+		
+		any_pop = {
+			ideology = { factor = 0.04 value = progressive }
+			ideology = { factor = 0.15 value = socialist }
+			ideology = { factor = 0.01 value = communist }
+		}
+		
+		ai_chance = { factor = 0.3 }
+	}
+	
+	option = {
+		name = "Democratic Centre (DC-PLI)"
+		ruling_party_ideology = conservative
+		
+		any_pop = {
+			ideology = { factor = 0.04 value = conservative }
+			ideology = { factor = 0.04 value = big_tent }
+		}
+		
+		ai_chance = { factor = 0.1 }
+	}
+	
+}
+
+#Elections 5
+country_event = {
+ 	id = 8020418
+  	title = "$YEAR$ Italian General Election"
+ 	desc = EVT_8020418_DESC
+  	picture = "ITA_1972_elections"
+
```

Comment:
This event must have fire_only_once = yes

Reply:
See above